resources/views/vendor/queue-monitor/jobs.blade.php

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>@lang('Queue Monitor')</title>
+
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
```

Comment:
Please remove external css resources.
Maybe we can implement this in out Adminpanel layout?

Reply:
That is a rather large can of worms. I'd like to wait with that until we're sure that this frontend is used (instead of grafana).